.github/workflows/tests.yml

```diff
@@ -0,0 +1,113 @@
+name: GitHub Actions CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  style:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Git repository
+        uses: actions/checkout@main
+
+      - name: Install shfmt
+        run: |
+          brew update
+          brew install shfmt
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@master
+        with:
+          bundler-cache: true
+
+      - run: script/style
+
+  tests:
+    runs-on: macos-12
```

Comment:
github actions support being run on a mac? neat.

Reply:
deleted the GitHub actions